app/views/shared/_navbar.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "#", class: "navbar-brand" do %>
-    <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <%= image_tag "https://res.cloudinary.com/agazielly/image/upload/v1566246427/joan-mm-EHuyA_oUCok-unsplash_ymnzy9.jpg" %>
```

Comment:
Is it a random picture ?

Reply:
No. I chose a picture from Unsplash website. I wanted it fixed.

Reply:
ok !